arch/preamble

```diff
@@ -100,6 +100,9 @@ NETCDF4_DEP_LIB = $(DEP_LIB_PATH) $(HDF5) $(ZLIB) $(GPFS) $(CURL)
 
 # NETCDF4INCLUDEGOESHERE
 
+#### CTSM pieces
+
+# CTSMINCLUDEGOESHERE
```

Comment:
@negin513 
Negin,
Is this used?

Reply:
Yes, this is used. I followed some other examples in this file. It is used by this line in Config.pl:

```perl
$_ =~ s:\# CTSMINCLUDEGOESHERE:include $sw_ctsm_mkfile_path: ;
```

Reply:
Thanks